static/src/javascripts/projects/common/modules/onward/popular.js

```diff
@@ -41,16 +42,10 @@ define([
         this.fetch(qwery('.js-popular-trails'), 'html');
     };
 
-    MostPopular.prototype.mobileMaximumSlotsReached = function () {
-        return (detect.getBreakpoint() === 'mobile' && $('.ad-slot--inline').length > 1);
-    };
-
     MostPopular.prototype.prerender = function () {
-        if (commercialFeatures.popularContentMPU && !this.mobileMaximumSlotsReached()) {
```

Comment:
A while ago I removed the limitation, but now that you mention it, that was for inline MPUs in article, not fronts. I'm putting this back. Thanks!